docs/workflows/publish-storybook.md

```diff
@@ -18,6 +18,17 @@ Storybook will create a static web application at the path you specify. This can
 npx http-server ./path/to/build
 ```
 
+<div class="aside">
+
+Asides from the `-o` flag, you can also include other flags to build Storybook, for instance if you're using [Docs](../writing-docs/introduction.md), you can append the `--docs` flag and Storybook will build your [MDX](../writing-docs/mdx.md) and [CSF](../writing-stories/introduction.md#component-story-format) stories into a rich and interactive documentation.
+
+You can learn more about these flag options [here](../api/cli-options.md).
+
+
+</div>
+
+
+If you need to build 
```

Comment:
Trailing sentence?

Reply:
That sentence wasn't meant to be there. I've removed it with the latest commit.